PERSISTENT-WORKERS.md

```diff
@@ -8,25 +8,21 @@ Follow the instruction in the [INSTALL.md](https://github.com/cirruslabs/cirrus-
 
 ## Running
 
-The most simplest invocation looks like this:
+The simplest invocation looks like this:
 
 ```
 cirrus worker run --token <poll registration token>
 ```
 
 This will start the persistent worker that periodically poll for new tasks in the foreground mode.
 
-By default the worker's name gets assigned semi-automatically based on the name of the current system and a counter variable. Specify `--name` to explicitly specify the worker's name:
```

Comment:
The `hostname` seems to be a bit ambiguous here: is it the command output, or a `hostname` string that will be used?

Maybe keep this "name of the current system"?

Reply:
See 39eb90ab15c08805013e1176803b44a9f4ea1669